mkt/websites/indexers.py

```diff
@@ -0,0 +1,89 @@
+from operator import attrgetter
+
+from mkt.search.indexers import BaseIndexer
+from mkt.translations.models import attach_trans_dict
+from mkt.translations.utils import format_translation_es
+
+
+class WebsiteIndexer(BaseIndexer):
+    translated_fields = ('description', 'short_title', 'title', 'url')
+
+    @classmethod
+    def get_mapping_type_name(cls):
+        return 'mkt_website'
+
+    @classmethod
+    def get_model(cls):
+        """Returns the Django model this MappingType relates to"""
+        from mkt.websites.models import Website
+        return Website
+
+    @classmethod
+    def get_mapping(cls):
+        """Returns an Elasticsearch mapping for this MappingType"""
+        doc_type = cls.get_mapping_type_name()
+
+        mapping = {
+            doc_type: {
+                'properties': {
+                    'id': {'type': 'long'},
+                    'category': cls.string_not_analyzed(),
+                    'created': {'type': 'date', 'format': 'dateOptionalTime'},
+                    'description': {'type': 'string',
+                                    'analyzer': 'default_icu'},
+                    'default_locale': cls.string_not_indexed(),
+                    'icon_hash': cls.string_not_analyzed(),
+                    'icon_type': cls.string_not_analyzed(),
+                    'last_updated': {'format': 'dateOptionalTime',
+                                     'type': 'date'},
+                    'modified': {'type': 'date', 'format': 'dateOptionalTime'},
+                    'short_title': {'type': 'string',
+                                    'analyzer': 'default_icu'},
+                    'title': {'type': 'string', 'analyzer': 'default_icu'},
+                    # FIXME: we probably want url to be analyzed, but with a
+                    # better suited analyzer.
+                    'url': {'type': 'string', 'analyzer': 'default_icu'},
```

Comment:
Possibly the simple analyzer? You can see how it will tokenize by hitting the URL: `http://localhost:9200/_analyze?text=http://www.example.com/test.html&analyzer=simple`. If not that maybe a custom analyzer that simply does a lowercase tokenizer. For URLs we might want to keep stopwords and not do stemming and such.